src/main/com/intellij/lang/jsgraphql/ide/project/GraphQLUIProjectService.java

```diff
@@ -371,6 +369,22 @@ public void run(@NotNull ProgressIndicator indicator) {
         }
     }
 
+    private void handleOperationName(GraphQLSettings graphQLSettings, GraphQLQueryContext context, Map<String, Object> requestData) {
```

Comment:
extracted to helper method, which handles getting the configured keyname for the operation name, and putting the correct value (either query|mutation|subscription name or "anonymous")